make/python-module-swig.mk

```diff
@@ -64,14 +64,14 @@ ifneq ($(SUBPROJECT_SO_FILES),)
 endif
 ifeq ($(OUTPUT_TYPE),kernel)
 	@echo '  PYTHON  $(PWD_REL_SEP)setup.py'
-	@cd $(BUILD_DIR) && python $(abspath setup.py) build
+	@cd $(BUILD_DIR) && EXT_SUFFIX='' python $(abspath setup.py) build -v
```

Comment:
is the change to this line necessary?

Reply:
I'm not actually sure--one sec, I'll remove it and if it still builds I'll push it up.

Reply:
Nope, it wasn't necessary!